cirq/ops/pauli_string_test.py

```diff
@@ -1676,13 +1677,18 @@ def test_circuit_diagram_info():
             1j * cirq.X(a) * cirq.Y(b),
             -1j * cirq.Y(b),
             1j**0.5 * cirq.X(a) * cirq.Y(b),
-        ), """
+        ),
+        """
```

Comment:
The trailing comma causes yapf to move the leading quotes to their own line, which for some reason seems to confuse pylint. I could only get it to ignore this long string literal by putting the `pylint: disable` directive at the top level. No idea why.

Reply:
Hmm, the battle of the "productivity" tools...well, soon enough yapf is going to go.